app/models/volunteer_op.rb

```diff
@@ -34,4 +42,16 @@ def self.contains_description?(text)
   def self.contains_title?(text)
     arel_table[:title].matches(text)
   end
+ 
+  def different_address?
```

Comment:
I'm wondering about this method name - isn't this checking that address and postcode are present, rather than whether the address is somehow different?

Perhaps this should be called `address_complete?`